gnosis/safe/safe.py

```diff
@@ -970,10 +973,12 @@ def retrieve_modules(
         self,
         pagination: Optional[int] = 50,
         block_identifier: Optional[BlockIdentifier] = "latest",
-    ) -> List[str]:
+        max_modules_to_retrieve: Optional[int] = 500,
```

Comment:
Could this limit affect us? I mean could we reach this limit in some cases?

Reply:
I don't think a Safe could have such a number of modules